src/Compilers/Core/MSBuildTaskTests/TargetTests.cs

```diff
@@ -515,6 +515,75 @@ public void AdditionalFilesAreAddedToNoneWhenCopied()
             Assert.Equal("Never", noneItems[2].GetMetadataValue("CopyToOutputDirectory"));
         }
 
+        [Theory, CombinatorialData]
+        [WorkItem(40926, "https://github.com/dotnet/roslyn/issues/40926")]
+        public void TestDisableAnalyzers(
+            [CombinatorialValues(true, false, null)] bool? runAnalyzers,
```

Comment:
💭 It shouldn't be necessary to specify `CombinatorialValues` in this case

Reply:
➡️ https://github.com/AArnott/Xunit.Combinatorial/issues/23

Reply:
Thanks!